src/lib.c

```diff
@@ -16,9 +16,9 @@
 
 #include "efivar.h"
 
-#define EFI_VARTOFILE	 "b2ac5fc9-92b7-4acd-aeac-11e818c3130c-VarToFile"
```

Comment:
You need a commit message that explains what's happening here

Reply:
Done